packages/fast-usdc/src/exos/settler.js

```diff
@@ -69,24 +78,20 @@ export const prepareSettler = (
       }),
       notify: M.interface('SettlerNotifyI', {
         notifyAdvancingResult: M.call(
-          M.record(), // XXX fill in details TODO
+          makeAdvanceDetailsShape(USDC),
```

Comment:
don't make the shape twice. Make it once in the top of `prepareSettler`.
```suggestion
          advanceDetailsShape,
```

Reply:
This actually only gets called once if we pull in the vstorage commit. I am going to pull that change into this PR and leave this as is.